.github/workflows/release_manager.yml

```diff
@@ -0,0 +1,70 @@
+name: Release Manager
+
+on:
+  push:
+    tags:
+      - "v[0-9]*"
+
+jobs:
+  release_manager:
+    name: Release Manager
+    runs-on: ${{ matrix.os }}
+    environment: Deploy Outline Manager
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-latest]
+      # Don't cancel jobs on other platforms if one fails
+      fail-fast: false
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v1
+
+      - name: Install Node.js, NPM and Yarn
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Show Environment Info
+        run: |
+          yarn -v
+          openssl version
+          env
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Check yarn cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: ${{ runner.os }}-yarn-
+
+      - name: Install dependencies
+        run: yarn --prefer-offline
+
+      - name: Release Linux
+        if: runner.os == 'Linux'
+        run: yarn do server_manager/electron_app/release_linux
+        env:
+          SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
```

Comment:
The Sentry DSN doesn't have to be a secret. Can we define a reusable variable here?

Reply:
Done in fd394edf5b81de46a06a4c0d7150f1ecaa4ea669